API/Controllers/PlayersController.cs

```diff
@@ -96,4 +96,22 @@ public async Task<IActionResult> ListAdminNotesAsync(int id)
 
         return Ok(await adminNoteService.ListAsync<PlayerAdminNote>(id));
     }
+
+    [HttpGet("{id:int}/friends")]
+    [Authorize(Roles = $"{OtrClaims.Roles.User}, {OtrClaims.Roles.Client}")]
```

Comment:
```suggestion
    [Authorize(Policy = AuthorizationPolicies.AccessUserResources]
```
I think to keep the expectation of privacy the same across data sources this needs to be only readable by the resource owner. You are not implicitly able to read the friends lists of others on osu! which is where this data is coming from in the first place.